src/components/Chat/MessageStage.tsx

```diff
@@ -87,9 +87,7 @@ export const MessageStage = ({ stage }: Props) => {
       {(stage.content || stage.attachments) && (
         <div
           className={`grid max-w-full grid-flow-row overflow-auto  ${
-            isOpened
-              ? 'border-t border-gray-400 p-2 dark:border-gray-700'
-              : 'h-0'
+            isOpened ? 'border-t border-primary p-2' : 'h-0'
```

Comment:
```suggestion
            isOpened ? 'border-t border-secondary p-2' : 'h-0'
```
![image](https://github.com/epam/ai-dial-chat/assets/20163971/6eda32ec-a0e1-40cd-b7fe-1243956a66e4)